build-tools/scripts/MonoAndroid.props

```diff
@@ -0,0 +1,29 @@
+<!--
```

Comment:
Could/should this instead modify [`build-tools/scripts/MonoAndroidFramework.props`](https://github.com/xamarin/xamarin-android/blob/c6b9a70b0c6d4f9fc4d66de28f3fff15b3531d67/build-tools/scripts/MonoAndroidFramework.props)?  The "rationale" seems to be the same.

Reply:
I moved the files around, and removed usage of the current `MonoAndroidFramework.props` from `Xamarin.Android.NUnitLite.NET.csproj`. It seems like this project doesn't need it at all?